lib/storage/file.js

```diff
@@ -902,14 +902,14 @@ File.prototype.getMetadata = function(callback) {
  *     members.
  *
  * @param {object} options - Configuration object.
- * @param {object} options.expiration - Timestamp (seconds since epoch) when
- *     this policy will expire.
  * @param {array|array[]=} options.equals - Array of request parameters and
  *     their expected value (e.g. [['$<field>', '<value>']]). Values are
  *     translated into equality constraints in the conditions field of the
  *     policy document (e.g. ['eq', '$<field>', '<value>']). If only one
  *     equality condition is to be specified, options.equals can be a one-
  *     dimensional array (e.g. ['$<field>', '<value>']).
+ * @param {object} options.expires - Timestamp (seconds since epoch) when this
```

Comment:
Why are we changing the verbiage? I know we're not `1.0.0` yet, but I'm not sure if this seems like something worth introducing a breaking change for?